src/librbd/Journal.h

```diff
@@ -11,6 +11,7 @@
 #include "common/Cond.h"
 #include "common/Mutex.h"
 #include "common/Cond.h"
+#include "common/WorkQueue.h"
```

Comment:
@badone Yes there is one but it is incomplete and we call its method in `Journal<I>::MetadataListener::handle_update()`. The recent compilers seem to be more restrictive towards this and do not allow incomplete declarations for this purpose.